### PR TITLE
Fixed #33754 -- Fixed crash with prematurely closed ASGI request body.

### PR DESCRIPTION
**Pull request by lundberg**
Ticket: https://code.djangoproject.com/ticket/33754

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -490,6 +490,7 @@ answer newbie questions, and generally made Django that much better:
     Jökull Sólberg Auðunsson <jokullsolberg@gmail.com>
     Jon Dufresne <jon.dufresne@gmail.com>
     Jonas Haag <jonas@lophus.org>
+    Jonas Lundberg <jonas.lundberg@gmail.com>
```
Comment (RhuanBorgesnr) [inline AUTHORS:493]: Nice

**Comment by felixxm** [comment]
@lundberg Thanks :+1: Welcome aboard :boat:

**Review by carltongibson — APPROVED** [review]
Thanks for this @lundberg — good catch. 👍 

Not for this PR but... _Ownership_ of the `body_file` gets passed to the request object in the `create_request()` call; ideally we'd encapsulate the clean up there... 🤔

**Comment by lundberg** [comment]
> Not for this PR but... _Ownership_ of the `body_file` gets passed to the request object in the `create_request()` call; ideally we'd encapsulate the clean up there... 🤔

Wouldn't we end up with the same problem if the `body_file` is closed in `create_request()`, i.e. the response haven't been processed yet and therefor touching `request.body` would still raise... 🤔

**Pull request merged**